contracts/core/RebalancingSetToken.sol

```diff
@@ -737,7 +703,7 @@ contract RebalancingSetToken is
      * Calcualate unit difference between both sets relative to the largest natural
      * unit of the two sets.
      */
-    function auctionSetUp()
+    function setUpAuctionDataStructures()
```

Comment:
setupAuctionParameters maybe? dataStructures is goofy

Reply:
+1 since it does more than just set up the arrays it also defines the minimumBid too.